yt_dlp/extractor/fujitv.py

```diff
@@ -16,20 +16,32 @@ class FujiTVFODPlus7IE(InfoExtractor):
     }
 
     _TESTS = [{
-        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810075',
+        'url': 'https://fod.fujitv.co.jp/title/5d40/5d40810081',
         'info_dict': {
-            'id': '5d40810075',
-            'title': '5d40810075',
+            'id': '5d40810081',
+            'title': '#1323 『まぼろしの「ツチノコ株式会社」』の巻',
+            'description': 'md5:c7b39106e5dd4891f67cc858585d1b0c',
+            'series_id': '5d40',
+            'series': 'ちびまる子ちゃん',
+            'thumbnail': 'http://i.fod.fujitv.co.jp/img/program/5d40/episode/5d40810081_a.jpg',
             'ext': 'mp4',
-            'format_id': '4000',
-            'thumbnail': 'http://i.fod.fujitv.co.jp/pc/image/wbtn/wbtn_5d40810075.jpg'
         },
-        'skip': 'Expires after a week'
+        'params': {
+            'skip_download': True
+        },
     }]
 
     def _real_extract(self, url):
-        video_id = self._match_id(url)
-        formats = self._extract_m3u8_formats(
+        series_id, video_id = self._match_valid_url(url).groups()
+        self._download_webpage(url, video_id)
+        json_info = {}
+        if self._get_cookies(url).get('CT'):
+            token = self._get_cookies(url).get('CT').value
+            json_info = self._download_json('https://fod-sp.fujitv.co.jp/apps/api/episode/detail/?ep_id=%s&is_premium=false' % video_id,
+                                            video_id, headers={'x-authorization': f'Bearer {token}'}, fatal=False)
+        else:
+            self.report_warning('Unable to extract token cookie, video information is unavailable')
```

Comment:
Add the "pass cookies to ..." message. You can use `self._LOGIN_HINTS['cookies']`

Reply:
should I put it in a new print statement or just in the warning statement?